### ISSUE TITLE
4.0 - Crash upon connecting a joypad

### ISSUE THREAD

**Issue by NoneGiven** [opentk/opentk#1091]
### Description

When connecting a Nintendo Switch Pro Controller to the computer via Bluetooth while an OpenTK `GameWindow` is already open, an exception occurs and crashes the process.

### Repro steps

1. Create a `GameWindow` and invoke its `Run` method.

2. Connect the controller (in my case, pressing any button on the controller connects it; it's already paired).

### Expected behavior

No issue.

### Actual behavior

Process crashes with the following exception:

```
Unhandled exception. System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at OpenToolkit.Windowing.Common.Input.JoystickState.SetButtonDown(Int32 index, Boolean value)
   at OpenToolkit.Windowing.Desktop.NativeWindow.ProcessInputEvents()
   at OpenToolkit.Windowing.Desktop.NativeWindow.ProcessEvents()
   at OpenToolkit.Windowing.Desktop.GameWindow.Run()
```

There is no crash if the controller is already connected before the `GameWindow` runs. I haven't verified whether this issue occurs with another kind of joypad.

### Related information 

* Windows 10
* NuGet package 4.0.0-pre9.1
* .NET Core 3.1

**Comment by NogginBops** [comment]
@NoneGiven Does the controller actually work if the controller is connected before opening the window?

**Comment by NoneGiven** [comment]
How can I test that? I'm not actually using controller input in my project, this is just something I came across by chance.

**Comment by NoneGiven** [comment]
Seems like #1095 might fix this.

**Comment by NoneGiven** [comment]
This is fixed in `4.0.0-pre9.2`.
